docs/docs/new-features.md

```
@@ -180,6 +180,38 @@ Everyone sees the margins and follows the presenter's point of view. If multi-us
+For an in-depth overview of this initiative, please refer to [issue 21059](https://github.com/bigbluebutton/bigbluebutton/issues/21059).
+Feature parity with the current media framework is not yet achieved, but the
+aforementioned issue provides parity tracking in section `Annex 1`.
+
+To enable support for LiveKit:
+  - Install bbb-livekit: `$ sudo apt-get install bbb-livekit`
+  - Enable the LiveKit controller module in bbb-webrtc-sfu: `$ yq e -i '.livekit.enabled = true`
+  - Restart bbb-webrtc-sfu: `$ sudo systemctl restart bbb-webrtc-sfu`
+
+Once enabled, LiveKit still won't be used by default. There's two ways to make
+use of it in meetings:
+  - Per meeting: set any of the following meeting `/create` parameters
```

Markdownlint (0.35.0)

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)